### DISCUSSION TITLE
Minimize to system tray / Close to system tray

### DISCUSSION THREAD

**Discussion by ozgurkazancci** [rustdesk/rustdesk#9454] [General]
Currently, RustDesk has to stay always open and/or minimized on taskbar.

I couldn't find an option such as; "Minimize to system tray / Close to system tray", to keep RustDesk running.

Whenever I click to 'X', RustDesk gets closed and no connection to this PC is possible, even though the service is still running.

Regards.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/8098
